sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/HelloWorldAsync.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics;
+
+import com.azure.ai.textanalytics.models.DetectedLanguage;
+
+public class HelloWorldAsync {
+
+    public static void main(String[] args) {
+        // Instantiate a client that will be used to call the service.
+        TextAnalyticsAsyncClient client = new TextAnalyticsClientBuilder()
+            .subscriptionKey("subscription-key")
+            .endpoint("https://servicename.cognitiveservices.azure.com/")
+            .buildAsyncClient();
+
+        // The text that need be analysed.
+        String text = "hello world";
+
+        client.detectLanguage(text).subscribe(
+            result -> {
+                final DetectedLanguage primaryLanguage = result.getPrimaryLanguage();
+                System.out.printf("Detected Language: %s, ISO 6391 Name: %s, Score: %s.%n",
+                    primaryLanguage.getName(), primaryLanguage.getIso6391Name(), primaryLanguage.getScore());
+            },
+            error -> System.err.println("There was an error detecting language of the text." + error),
+            () -> System.out.println("Language detected."));
```

Comment:
this program will end before any of those call backs are invoked. for the same of the sample, there should be a note about this.

https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/eventhubs/azure-messaging-eventhubs/src/samples/java/com/azure/messaging/eventhubs/PublishEventsToSpecificPartition.java#L96